central/cve/image/info/datastore/store/store.go

```diff
@@ -0,0 +1,25 @@
+package store
+
+import (
+	"context"
+
+	v1 "github.com/stackrox/rox/generated/api/v1"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/search"
+)
+
+// Store provides storage functionality for ImageCVEInfos.
+//
+//go:generate mockgen-wrapper
```

Comment:
Same question as datastore.  If we aren't using it or going to, no point to generate it.